src/ui/settings/ReleaseNotes.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": [
+    {
+      "version": "unreleased-alpha",
```

Comment:
Are you changing this in the future? What I do is I set the next version here so I don't forget before releasing.

Reply:
I usually set it when I bump for release depending on what has been done - so next could be either 3.5.0 or 3.4.1.